ux.md

```diff
@@ -3,29 +3,110 @@
 3. An icon appears in their System Tray (top right on macOS, bottom right on 
 Windows), with the a mini Node Launcher logo and an orange status dot. If the 
 user clicks on it, they see a progress bar with "Syncing with the Lightning 
 Network..." What the Node Launcher is doing in the background is using neutrino 
-"SPV" to quickly sync Lightning, and launches bitcoind in the background so 
-they can start syncing their full node. When the neutrino sync is done, the 
-status dot becomes blue. When the bitcoind full node sync is done, LND connects 
-to it directly and the status dot becomes green. Both nodes default to using 
-Tor.
+"SPV" to quickly sync Lightning. A notification is shown to the user requesting
+him to choose between pruning and non-pruning mode. Pruning is the default,
+recommended setting. Non-pruning is disbled if the user has less than 300GB of
+free disk space. A compuer parameter scan is started in parallel to determine
+the best configuration for dbcache etc. Once the user confirms the setting
+bitcoind is launched in the background so they can start syncing their full node.
+When the neutrino sync is done, the status dot becomes blue. When the bitcoind
+full node sync is done, LND connects to it directly and the status dot becomes
+green. Both nodes default to using Tor.
 4. User receives a Node Launcher desktop notification to install the Joule 
 browser extension, with a link to https://lightningjoule.com/
 5. User installs Joule, and Joule automatically connects to the local LND node 
 without any manual configuration at all
 6. User receives a Node Launcher desktop notification to deposit BTC in their 
-LND wallet, clicking it pops up a QR code  
+LND wallet, clicking it pops up a QR code. The notification has orange warning
+that it's currently unsuitable for large amounts. After clicking more info, the
+user is informed that they should wait for sync to be finished and to purchase HW
+wallet for savings. Additionally a checkbox "Unlink bitcoins" is present and
+checked by default. Wasabi or Joinmarket tool is used in the background to unlink
+the deposited bitcoins before opening the channels if the checkbox is checekd.
 7. User deposits BTC, channels automatically open
 8. Node Launcher communicates estimated time before channels are open
-9. User can now send satoshis with Joule!
\ No newline at end of file
+9. User can now send satoshis with Joule!
+
+## Upgrade process
+
+1. If the package is installed using a package manager, automatic updates for the
+package are disabled by default using post-install script in order to prevent
+consensus attacks.
+2. When a new update is available, the user is informed about it and a window asking
+confirmation appears. The window contains information about how many developers
+of the launcher signed that the upgrade is OK and what is the total number of
+developers needed. If the number is lower than half, the update is ignored. User can
```

Comment:
This part I'm hesitant about, but we can leave as-is until we get closer to implementation. I'd prefer just an external link with release notes for the update

Reply:
The reason behind it is to prevent situation where the attacker compromises a website (or a developer) and leaves out information about consensus changes.